sr_robot_launch/launch/sr_right_ur10arm_hand.launch

```diff
@@ -27,6 +27,10 @@
   <arg name="robot_description" default="$(find sr_multi_description)/urdf/right_srhand_ur10_joint_limited.urdf.xacro"/>
   <arg name="robot_config_file" default="$(find sr_multi_moveit_config)/config/robot_configs/right_sh_ur10.yaml"/>
 
+  <!-- When true, robot_state_publisher publishes static frames to /tf_static. This can cause problems with the current rospy tf API,
+       which can't see /tf_static due to a bug (that will be fixed by https://github.com/ros/geometry/pull/134 once merged). -->
```

Comment:
Sadly this is unnecessary. 
The source of the problem is that you are trying to use tf instead of using tf2 (the new library that suppors /tf_static).
Again this is probably the very common issue of obsolete documentation in important pages of the ROS wiki.
That is, landing [here](http://wiki.ros.org/tf/Tutorials/Writing%20a%20tf%20listener%20%28Python%29) instead of [here](http://wiki.ros.org/tf2/Tutorials/Writing%20a%20tf2%20listener%20%28Python%29)